### PR TITLE
Use the dos path plus the optional `fileName` for localization, plus... WA-347

### PR DESCRIPTION
**Pull request by kshakir**
- Use the response itself to determine if the GCF was speaking martha_v2 or martha_v3 syntax
- Local testing updates incl. allowing CROMWELL_BUILD_GENERATE_COVERAGE to be pre-est
- Updated the unused DOS/DRS time converter to parse both w/ & w/o TZ

### CODE REVIEW

**Comment by kshakir** [comment]
NOTE: Centaur tests require https://github.com/broadinstitute/martha/pull/186 merged-and-deployed

**Comment by kshakir** [comment]
See comments in WA-344. While all that is discussed I'm icing this pull request.

**Comment by kshakir** [comment]
Tests will fail until review/merge/release of https://github.com/broadinstitute/martha/pull/190

**Comment by kshakir** [comment]
FYI ready for 👀 @rsasch and @salonishah11. Regarding the failing dbms test: it's not failing due to this PR's changes and is not a blocker for review.

I suspect that failure has something to do with lucky number [PostgreSQL 13](https://news.ycombinator.com/item?id=24578166) coming out yesterday. When @cjllanwarne  finishes fighting Prod fires he (or perhaps someone else?) might have a small PR that can fix the issue in develop. When that fix merges I'll just restart the test in this PR and it should pick up the develop changes automagically and pass.

**Review by rsasch — APPROVED** [review]

File: cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala [hunk 1]
```diff
@@ -135,6 +150,7 @@ object MarthaResponseSupport {
       name = fileName,
       gsUri = gcsUrl,
       googleServiceAccount = response.googleServiceAccount,
+      fileName = None,
```
Comment (salonishah11) [inline cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala:153]: Why not extract the `name` key from Martha V2 response [here](https://github.com/broadinstitute/cromwell/blob/222976d88c53d451c7d68fae46ffada9451e6da2/cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala#L73-L76) and pass it along as `fileName` here?

**Review by salonishah11 — APPROVED** [review]

**Pull request merged**